src/main/java/com/stemcraft/feature/SMChatFormat.java

```diff
@@ -65,7 +65,7 @@ protected Boolean onEnable() {
                 format = replaceAll(PREFIX_PLACEHOLDER_PATTERN, format, () -> colorize(this.vaultChat.getPlayerPrefix(event.getPlayer())));
                 format = replaceAll(SUFFIX_PLACEHOLDER_PATTERN, format, () -> colorize(this.vaultChat.getPlayerSuffix(event.getPlayer())));
             } else {
-                System.out.println("vault null");
+                this.plugin.getLogger().warning("vault null");
```

Comment:
This whole else block can go. It is a debug message. We should never reach here based on line 43